randomgift-sheet.gs

```diff
@@ -1,27 +1,27 @@
 // EN locale
 //////////////////////////////////////
 var msgUIEN = {
-  "sent_title": "Success: All emails sent",
-  "sent": "Congratulation, the distribution has been successful and all e-mails have been sent. Enjoy your Event!",
-  "error": "Sorry, something went wrong."
+  "sent_title": "Exito: Todos los correos enviados",
+  "sent": "Felicidades, se ha completado la distribución y todos los correos han sido enviados. Disfruta tu evento!",
+  "error": "Lo siento, algo salió mal."
 }
 
 var msgMailEN = {
-  "dear": "Dear",
-  "participants": "The people participating are",
-  "signature": "Enjoy!",
-  "target": "is your target for this event.",
+  "dear": "Estimad@",
```

Comment:
Are you using a "@" in order to stay neutral?
Is this something that is common? what about using "Estimada / Estimado"?

Reply:
Yes, I'm using @ to stay neutral... It's not a formal use, but it's common and a lot shorter (and friendlier) than Estimado / Estimada...